ui/page/channel/view.jsx

```diff
@@ -250,7 +250,15 @@ function ChannelPage(props: Props) {
         <TabPanels>
           <TabPanel>
             {searchResults ? (
-              searchResults.map(url => <ClaimPreview key={url} uri={url} />)
+              <ClaimList
+                header={false}
+                headerAltControls={null}
+                id={`search-results-for-${claimId}`}
+                loading={false}
+                showHiddenByUser={false}
+                type={''}
```

Comment:
@seanyesmunt please review these parameters. For 'type' I saw that some options are 'small, 'large', 'inline', etc.. All of these options gave undesired results. What value should I use here if I just want the default behavior?

Reply:
@dalhill you can just remove that line and it will use the default.